.gitignore

```diff
@@ -1,3 +1,9 @@
 # IDE stuff
 .vscode
+__debug*
+__DEBUG*
+__Debug*
 
+# local dev
+.env
```

Comment:
might want to include this file later. but for now best to keep it out of commit history.